commitlintplugins.test.ts

```diff
@@ -160,6 +160,24 @@ test('body-max-line-length7', () => {
 });
 
 
+test('commit-hash-alone1', () => {
+    let commitMsgWithCommitUrl= 
+        "foo: this is only a title" + "\n\n" + 
+        "https://github.com/nblockchain/conventions/commit/3ee07243edc30604088a4b04ca525204ea440710";
+    let commitHashAlone1 = runCommitLintOnMsg(commitMsgWithCommitUrl);
+    expect(commitHashAlone1.status).not.toBe(0);
+});
+
+
+test('commit-hash-alone2', () => {
+    let commitMsgWithCommitHash = 
+        "foo: this is only a title" + "\n\n" + 
+        "This is refering to [1] commit hash.\n\n[1] 3ee07243edc30604088a4b04ca525204ea440710";
```

Comment:
2 things here:

- typo "refering" -> "referring"
- if you're using [1], then it should be there twice, first where it's used, and at the end as a footer note (if you don't understand this, ask Parham because he recently fixed a unit test that had the same problem)

Reply:
oops! nevermind, I didn't see the second [1], scratch that